share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -209,9 +209,9 @@ A Hash specifies a formula dependency with some additional information. Given a
 
 Sometimes there’s hard conflict between formulae, and it can’t be avoided or circumvented with `keg_only`.
 
-PolarSSL is a good [example](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/polarssl.rb) formula for minor conflict.
+Mbed TLS is a good [example](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mbedtls.rb) formula for minor conflict.
 
-PolarSSL ship and compile a "Hello World" executable. This is obviously non-essential to PolarSSL’s functionality, and conflict with the popular GNU `hello` formula would be overkill, so we just remove it.
+Mbed TLS ships and compile a "Hello World" executable. This is obviously non-essential to mbed’s functionality, and conflicts with the popular GNU `hello` formula would be overkill, so we just remove it.
```

Comment:
And `conflict` shouldn't be pluralised in this context. It is intended to read *"and conflict with the popular GNU `hello` formula would be overkill"* rather than *"and conflicts with the popular GNU..."*